src/components/atoms/ActiveCount/ActiveCount.tsx

```diff
@@ -1,9 +1,12 @@
 interface ActiveCountProps {
   count?: number;
+  className?: string;
 }
 
-const ActiveCount = ({ count }: ActiveCountProps) => {
-  return <p className="ml-3 font-light text-main-300">{count}</p>;
+const ActiveCount = ({ count, className }: ActiveCountProps) => {
+  return (
+    <p className={`ml-3 font-light text-main-300 px-1 ${className}`}>{count}</p>
```

Comment:
Co to jest className?

Reply:
Dodatkowe klasy przekazywane jako parametr. ActiveCount jest używany i w listach predefiniowanych i w listach użytkownika, a w listach użytkownika ma dodatkową klasę

Reply:
a po co są te klasy? lepiej mieć jakiś flagi i według nich ustawiać odpowiednie style w komponencie styled komponent